conferences/2024-09-18__LPC-UAPI-MC.md

```diff
@@ -0,0 +1,40 @@
+# Kernel <-> Userspace/Init/System Management boundaries and APIs MicroConference @ LPC 24
+
+We are pleased to announce the Call for Participation (CfP) for the [LPC 2024](https://lpc.events/)
+Kernel <-> Userspace/Init/System Management boundaries and APIs MicroConference.
+
+LPC 2024 will be held in Vienna, Austria, September 18th to 20th 2024.
+FOSDEM is a free and non-commercial event organised by the community for the community.
```

Comment:
```suggestion
FOSDEM is a free and non-commercial event organised by the community for the community.
```

This should be removed.

Reply:
I wonder how that ended up there - I absolutely, most definitely did not copy and paste the CFP from last time

Reply:
I'm 99% sure you just asked Copilot for a CFP :D